ansible/playbooks/etcd-setup.yml

```diff
@@ -0,0 +1,10 @@
+---
```

Comment:
I usually make the main playbook idem-potent, doing everything which makes sense at least. So you can run it again and again (= deploy + configure). Restarts are done with notifiers (compare https://github.com/sttts/compute-platform/blob/master/roles/mesos/tasks/master.yml#L8).

Those steps which do not fit in this scheme (or are dangerous to do automatically), I split them out in special playbooks.